frontend/src/packages/editor/components/EditorBlock.tsx

```diff
@@ -73,9 +120,17 @@ const EditorBlock: FC<CMSBlockProps> = ({
         code: leaf.code ?? false,
         align: leaf.align ?? 'left',
         textSize: leaf.textSize ?? defaultTextSize,
+        checklist: leaf.checklist ?? false,
+        checked: leaf.checklist ?? false,
```

Comment:
This doesn't appear to be used elsewhere so you can remove it. Let me know if i missed its use case